py3dtilers/Common/tileset_creation.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+from py3dtiles import B3dm, BatchTable, BoundingVolumeBox, GlTF
+from py3dtiles import Tile, TileSet
+from ..Common import create_lod_tree
+from ..Common import createTextureAtlas
+
+
+def create_tileset(objects_to_tile, also_create_lod1=False, also_create_loa=False, loa_path=None, extension_name=None, with_texture=False):
```

Comment:
Same question as above concerning the regrouping of loose functions to some class...